products/azure_rm_servicebusdisasterrecoveryconfig/api.yaml

```diff
@@ -0,0 +1,67 @@
+--- !ruby/object:Api::Product
+name: Azure Automation Variable
+prefix: arm
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: NotUsedInAzure
+scopes:
+  - NotUsedInAzure
+azure_namespace: 'Microsoft.Automation'
+azure_version: 'stable/2015-10-31'
+objects:
+  - !ruby/object:Api::Resource
+    name: AutomationVariable
+    api_name: AutomationVariable
+    base_url: NotUsedInAzure
+
+    azure_sdk_definition: !ruby/object:Api::Azure::SDKDefinition
+      provider_name: Microsoft.Automation
+      create: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: CreateOrUpdate
+        request:

```

Comment:
**Must Fix** Request is required when generating code. Take an example here: https://github.com/VSChina/magic-modules/blob/azure_backend/products/azservicebusrecovery/api.yaml#L24-L43.

Typically in the request, `go_type_name`/`go_field_name` is the field name in Go SDK (which is also generated from Autorest). `go_variable_name` has some convention, for example, `resource_group_name` is always `resourceGroup`, resource name is always `name`, and request body object `/` is always `parameters`.